imageio/core/format.py

```diff
@@ -631,7 +631,18 @@ class FormatManager(object):
 
     @property
     def _formats(self):
-        return [x for x in known_plugins.values() if x.is_legacy]
+        available_formats = list()
+
+        for config in known_plugins.values():
+            if config.is_legacy:
+                try:
+                    config.format
+                except ImportError:
+                    pass  # format not installed
+                else:
+                    available_formats.append(config)
```

Comment:
```suggestion
            with contextlib.suppress(ImportError):
                # if an exception is raised, then format not installed
                if config.is_legacy and config.format is not None:                    
                    available_formats.append(config)
```

this requires adding `import contextlib` at the beginning of the file, but for me, it is closer.

Reply:
@Czaki Nice. Learned about a handy new function today, thanks for the pointer :)